frontend/src/pages/about/hmi.js

```diff
@@ -88,7 +88,26 @@ const HmiPage = ({ pageContext }) => {
         <StaticImage src="../../images/flag-jordan.png" alt="Flag of Jordan" />
       ),
     },
-  ];
+  ].sort((a, b) => {
+    const recentYearsAlpha = a.year.split(/,\s*/g).pop();
```

Comment:
nit:refactor this into a function that returns a `Number` so you can reuse the function and also not need to keep converting to Number